src/style.rs

```diff
@@ -13,6 +13,12 @@ pub enum OutputComponent {
     Plain,
 }
 
+#[derive(Debug, Eq, PartialEq, Copy, Clone, Hash)]
+pub enum OutputWrap {
```

Comment:
since there are only two variants, each with no data, maybe this can be represented with a `bool` instead

Reply:
I used an enum in case anybody wanted to add different wrapping styles (e.g. word wrap) in the future, but I could change it to a bool if that's preferable.

Reply:
Oh right, I guess you can just go ahead and add that variant then

Reply:
I think I prefer the enum here, even if there are just two variants for now ("Boolean blindness"). We shouldn't add the third variant if it is not yet used.